package.json

```diff
@@ -22,7 +22,7 @@
   "dependencies": {
     "@avcs/autosuggest": "^1.7.5",
     "@floip/expression-evaluator": "^2.1.0",
-    "@floip/flow-runner": "https://github.com/FLOIP/flow-runner.git#feature/VMO-4281-flow-spec-rc2",
+    "@floip/flow-runner": "https://github.com/FLOIP/flow-runner.git#feature/VMO-4281-flow-spec-rc2#fdf532cf413fc363e5413f0013e2f86138127499",
```

Comment:
I prefer to keep the branch name as possible, because it gives us info that we're not consuming `master` yet

Reply:
Hi @safydy, sorry for so much discussion. I like your point of retaining the branch name 🙆‍♀️ !

From what I understand, this commit SHA _might_ be disregarded by yarn when using `https://`. Can you confirm that you trust that if we change this again, it will update `yarn.lock`? Thanks so much for being on top of this.

Please note :We don't have to solve for this right now, if it's easier to continue using yarn.lock for now, that's cool too!

Reply:
Hey @bzabos , I'm not confident our CI will have authorization with `git` protocol. 

For `https`,  I think you make a point here about yarn could be tricky.  I tested with another hash `88858a2baf5889ece8f9abbd575bdc92adfbe4a4`. Here are the results:
**[keep branch name & hash]**
package.json
`https://github.com/FLOIP/flow-runner.git#88858a2baf5889ece8f9abbd575bdc92adfbe4a4
"@floip/flow-runner": "https://github.com/FLOIP/flow-runner.git#feature/VMO-4281-flow-spec-rc2#88858a2baf5889ece8f9abbd575bdc92adfbe4a4",`

after yarn install, yarn.lock wasn't full updated. Only the key has been update
`"@floip/flow-runner@https://github.com/FLOIP/flow-runner.git#feature/VMO-4281-flow-spec-rc2#88858a2baf5889ece8f9abbd575bdc92adfbe4a4":`
  
But `version` & `resovled` were not. I assume this is the behaviour if I downgrade it by keeping branch name & hash 
```
  version "1.0.0-rc2.6"
  resolved "https://github.com/FLOIP/flow-runner.git#fdf532cf413fc363e5413f0013e2f86138127499"
```

**[with hash only]**
`package.json
https://github.com/FLOIP/flow-runner.git#88858a2baf5889ece8f9abbd575bdc92adfbe4a4`

after yarn install, got yarn.lock fully updated with
```
"@floip/flow-runner@https://github.com/FLOIP/flow-runner.git#88858a2baf5889ece8f9abbd575bdc92adfbe4a4":
  version "1.0.0-rc2.5"
  resolved "https://github.com/FLOIP/flow-runner.git#88858a2baf5889ece8f9abbd575bdc92adfbe4a4"
 ```

So if you're ok, for now, I'd suggest we stick with:
- providing branch info in `package.json` eg: `"@floip/flow-runner": "https://github.com/FLOIP/flow-runner.git#feature/VMO-4281-flow-spec-rc2`
- specify manually the hash in `yarn.lock`

Reply:
@jcbashdown , we just need to resolve this discussion and we're good I think